deployment/ansible/deploy.yml

```diff
@@ -9,7 +9,7 @@
     config:
       chain:
         persist: true
-        db_path: "./db4"
+        db_path: "./db6"
```

Comment:
Depending on which PR is merged first, please double check this value in the other PR.